pkg/kubelet/types/constants.go

```diff
@@ -21,3 +21,21 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// HairpinMode denotes how the kubelet should configure networking to handle
+// hairpin packets.
+type HairpinMode string
+
+// Enum settings for different ways to handle hairpin packets.
+const (
+	// Set the hairpin flag on the veth of containers in the respective
+	// container runtime.
+	HairpinVeth = "hairpin-veth"
+	// Make the container bridge promiscuous. This will force it to accept
+	// hairpin packets, even if the flag isn't set on ports of the bridge.
+	HairpinBridge = "promiscuous-bridge"
```

Comment:
Sure i can rename it tomorrow, security key stopped working so i don't have remote access.

HairpinMode is a type, so you don't have to access it as hairpinmode.Promiscuous bridge. The HairpinMode type is just used for clarity in struct fields, func args etc (instead of a naked string), and we'd access it as kubetypes.HairpinBridge.